lemming.go

```diff
@@ -176,6 +182,15 @@ func WithSysribAddr(sysribAddr string) Option {
 	}
 }
 
+// WithNoRIBForwardReferences specifies that the lemming's RIB should not
+// allow forward references - e.g., a next-hop-group that references a next-hop
+// that does not exist.
+func WithNoRIBForwardReferences() Option {
```

Comment:
if we want to expose more gribi server options, we could do WithGRIBIOptions(gribis.Option)